core/src/test/java/com/radixdlt/rev2/protocol/AnemoneProtocolUpdateTest.java

```diff
@@ -205,96 +194,37 @@ public void test_get_current_time_second_precision() {
 
   @Test
   public void core_api_streams_anemone_flash_transactions() throws Exception {
-    final var coreApiPort = FreePortFinder.findFreeLocalPort();
-    try (var test = createTest(createCoreApiModule(coreApiPort))) {
+    final var coreApiHelper = new ProtocolUpdateTestUtils.CoreApiHelper();
+    try (var test = createTest(coreApiHelper.module())) {
```

Comment:
Arguably this might be easier as a Core API integration test, but this is fine too - thanks for pulling out the Core API helper to make this at least a little neater!

Reply:
Totally agree :D. Good use of composition over inheritance!